lisa/testsuite.py

```diff
@@ -107,6 +107,26 @@ def _send_result_message(self) -> None:
             )
             result_message.environment_information.update(environment_information)
             result_message.environment_information["name"] = self.environment.name
+            platform = list(
+                self.environment.default_node.capability.extended_schemas.keys()
+            )[0]
+            result_message.environment_information["platform"] = platform
+            # get image type, e.g. gallery
+            image_type = self.environment.default_node.capability.extended_schemas[
```

Comment:
image_type is not shared information for all platform. So this logic should be in platform information hook.